__tests__/app.test.js

```diff
@@ -132,10 +132,19 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
   test("404:responds with an error, this article_id does not exist yet", () => {
     return request(app)
-      .get('/api/articles/19/comments')
+      .get('/api/articles/100/comments')
       .expect(404)
         .then((response) => {
-          expect(response.body.msg).toBe("this id does not exist yet");
+          expect(response.body.msg).toBe("this article does not exist yet");
         });
     });
 });
+test("404:responds with an error, this article_id does not have any comments", () => {
+  return request(app)
+    .get('/api/articles/2/comments')
+    .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("this article does not have any comments");
+      });
+  });
```

Comment:
This is a good start, but it should be a successful response of 200 with an empty array as its response. The reason for this is that there is nothing wrong with not having any attached comments and an output of an empty array would then be expected.